aiu_fms_testing_utils/utils/paged.py

```diff
@@ -328,16 +329,16 @@ def generate(
                         if chunk_j == 0:
                             chunk_start = 0
                             chunk_end = prefill_chunk_size - required_extra_pads
-                        else:
+                        else: 
                             required_extra_pads = 0
                             chunk_start = chunk_end
                             chunk_end += prefill_chunk_size
 
-                        input_ids_seq_chunk = input_ids[seq_i][chunk_start:chunk_end]
-                        slot_mapping_seq_chunk = slot_mapping[seq_i][
+                        input_ids_seq_chunk = input_ids[seq_i][-current_tkv:][chunk_start:chunk_end]
```

Comment:
this was needed to get the correct ids besides pad tokens . we pad the chunk again after this.